src/catalog/protos/objects.proto

```diff
@@ -22,767 +22,772 @@ syntax = "proto3";
 package objects;
 
 message ConfigKey {
-    string key = 1;
+  string key = 1;
 }
 
 message ConfigValue {
-    uint64 value = 1;
+  uint64 value = 1;
 }
 
 message SettingKey {
-    string name = 1;
+  string name = 1;
 }
 
 message SettingValue {
-    string value = 1;
+  string value = 1;
 }
 
 message IdAllocKey {
-    string name = 1;
+  string name = 1;
 }
 
 message IdAllocValue {
-    uint64 next_id = 1;
+  uint64 next_id = 1;
 }
 
 message GidMappingKey {
-    string schema_name = 1;
-    CatalogItemType object_type = 2;
-    string object_name = 3;
+  string schema_name = 1;
+  CatalogItemType object_type = 2;
+  string object_name = 3;
 }
 
 message GidMappingValue {
-    uint64 id = 1;
-    string fingerprint = 2;
+  uint64 id = 1;
+  string fingerprint = 2;
 }
 
 message ClusterKey {
-    ClusterId id = 1;
+  ClusterId id = 1;
 }
 
 message ClusterValue {
-    reserved 2;
-    string name = 1;
-    RoleId owner_id = 3;
-    repeated MzAclItem privileges = 4;
-    ClusterConfig config = 5;
+  reserved 2;
+  string name = 1;
+  RoleId owner_id = 3;
+  repeated MzAclItem privileges = 4;
+  ClusterConfig config = 5;
 }
 
 message ClusterIntrospectionSourceIndexKey {
-    ClusterId cluster_id = 1;
-    string name = 2;
+  ClusterId cluster_id = 1;
+  string name = 2;
 }
 
 message ClusterIntrospectionSourceIndexValue {
```

Comment:
Not sure how `ClusterIntrospectionSourceIndexValue` is used, but it seems like not all objects, e.g. cluster and replicas, get oids?

Reply:
`OID`s and their usages mainly exist for PG compatibility instead of a first class identifier for objects. As a result, we decided not to give non-PG objects (e.g. clusters and replicas) OIDs. `ClusterIntrospectionSourceIndexValue`s are treated as normal(ish) indexes so they get OIDs. We can revisit this in the future, but I don't think we should change it in this PR.